homeassistant/components/yalexs_ble/strings.json

```diff
@@ -0,0 +1,30 @@
+{
+  "config": {
+    "flow_title": "{name} ({local_name}) {address}",
```

Comment:
do we need 2 names? And do we still care about the address if we already have a name?

Reply:
Some of the older firmwares do not have a unique local name and present as 'Yale' 'Aug' or 'Zeyphr' so it's hard to tell the locks apart. Especially on macOS where the address is a uuid.  I think we can reduce the address to the last 4 like we do in switchbot. That would avoid the long string

I'll add a utility function to figure out what a good name is and reduce it down to one name.

Reply:
f4b011201bdaaa38aab8a6add6acdc85fe08373a